junit5/src/main/java/com/linecorp/armeria/testing/junit5/client/HttpHeadersAssert.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.testing.junit5.client;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.linecorp.armeria.common.ContentDisposition;
+import com.linecorp.armeria.common.HttpHeaders;
+import com.linecorp.armeria.common.MediaType;
+
+import io.netty.util.AsciiString;
+
+/**
+ * Assertion methods for HttpData.
+ */
+public final class HttpHeadersAssert extends AbstractResponseAssert<HttpHeaders> {
+    HttpHeadersAssert(HttpHeaders actual, TestHttpResponse response) {
+        super(actual, response);
+    }
+
+    /**
+     * Asserts that the content length of actual {@link HttpHeaders} is equal to the given one.
+     */
+    public TestHttpResponse contentLengthIsEqualTo(long expected) {
```

Comment:
Suggestion; would the following be more natural?

```suggestion
    public TestHttpResponse hasContentLength(long expected) {
```